_posts/2024-10-17-query-insights.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: "Gain deeper insights with OpenSearch Query Insights" 
+authors:
+   - chenyang
+   - kolchfa
+date: 2024-10-17 
+categories:
+  - technical-posts
+meta_keywords: opensearch query insights, search performance, query optimization, search analytics, resource monitoring, latency analysis, cpu usage, memory consumption, troubleshooting, search visibility, metrics, dashboards
+meta_description: Gain deeper visibility into search query performance with OpenSearch Query Insights. This tool helps identify bottlenecks, optimize resources, and troubleshoot issues by providing detailed insights into query execution, latency, CPU/memory usage, and more. Enhance your search analytics and deliver a better user experience.
+---
```

Comment:
Please update the meta with: 

**_meta_keywords:_** OpenSearch query insights, search performance, query optimization, query latency, search analytics, resource monitoring

_**meta_description:**_ The query insights feature in OpenSearch gives you visibility into search performance, enabling you to identify bottlenecks, optimize resources, and enhance troubleshooting to deliver a better search experience for users.